### DISCUSSION TITLE
Hook failed on Vue 3 - TypeScript

### DISCUSSION THREAD

**Discussion by turkerdev** [tailwindlabs/tailwindcss#3826] [Help] [answered]
vue add tailwindcss@latest
 WARN  There are uncommitted changes in the current repository, it's recommended to commit or stash them first.
? Still proceed? Yes
📦  Installing vue-cli-plugin-tailwindcss...
up to date, audited 1538 packages in 3s
88 packages are looking for funding
  run `npm fund` for details
found 0 vulnerabilities
✔  Successfully installed plugin: vue-cli-plugin-tailwindcss
🚀  Invoking generator for vue-cli-plugin-tailwindcss...
⚓  Running completion hooks...
 ERROR  Error: ENOENT: no such file or directory, open 'C:\Users\bajosi\Desktop\project\client\src\main.js'

**Comment by simonswiss** [comment, marked as answer]
Hey!

Looks like this is a question for the [Vue CLI](https://cli.vuejs.org/) rather than Tailwind CSS?

Unless you meant to type `yarn add tailwindcss@latest` instead of `vue add tailwindcss@latest` in the first line?
